config/co2eq_parameters.json

```diff
@@ -326,6 +326,10 @@
         "hydro discharge": {
           "source": "2019 average by Tomorrow",
           "value": 48.14997796838915
+        },
+        "nuclear": {
+        "source": "ADEME",
+        "value": 6
         }
```

Comment:
```suggestion
        "nuclear": {
          "_url": "https://www.bilans-ges.ademe.fr/documentation/UPLOAD_DOC_FR/index.htm?conventionnel.htm"
          "source": "ADEME 2015",
          "value": 6
        }
```

Reply:
Ok, it is not as if there was an emergency on the matter, so it's quite fine.

For my curiosity what do you call "your pipeline" ?

I saw there was a pull request for a regional parser in France, do the regional override work with hereditary (FR_XXX inherits from FR unless an override is defined for FR_XXX) ?

Thanks  best regards